pages/blog/asynchronously-exporting-data-to-remote-systems.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: "Asynchronously Exporting Data to Remote Systems"
+date: "2021-05-27"
+author: "Pedro S Lopez"
+pullQuote: "Some batch import APIs work asynchronously and don't process your requests right away."
+image: "asynchronously-exporting-data-to-remote-systems/bus-stop.jpeg"
+tags: [engineering]
+---
+
+When integrating with destinations, there are generally two main approaches made available by API providers: **single** or **batched**. With the "single" approach, one API request usually affects a single profile in the destination. The "batched" approach, which you can read more about [here](https://www.grouparoo.com/blog/batching-api-requests), allows you to affect multiple profiles in a single API request. This is usually preferred due to rate limits put in place and because they are often faster than their single counterparts when dealing with a large number of profiles.
+
+However, within batched APIs you may find a few different approaches. In most cases, issuing an API request to import a batch of profiles to a destination resolves right away and immediately gives a response with information about whether the action was successful or not, as well as which profiles had errors. On the other side, some APIs enqueue the action to be processed later and instead require you to check back on its status to know if it was successful.
+
+## Working with Asynchronous Destinations
```

Comment:
Any insight on why a provider might choose to implement their API with an Asynchronous Destinations?